### ISSUE TITLE
bad result

### ISSUE THREAD

**Issue by degilala** [uncbiag/mermaid#151]
I used this code to train my own data set. My data set came from ADNI. I used 240 pieces as the training set, 70 pieces as the verification set, and 34 pieces as the test set.I did not change other parameters, but the result of training test is worse than Voxelmorph. Could you please give me some opinion analysis.

**Comment by marcniethammer** [comment]
I am not sure what you are referring to. Mermaid does not do deep-learning based registration. 
If you are looking for deep-learning-based registration our currently most actively developed codebase is here: https://github.com/uncbiag/ICON

**Comment by degilala** [comment]
sorry，I didn't make myself clear.I used the code demo_for_easyreg_train.py from  https://github.com/uncbiag/easyreg to train my model.
